src/Microsoft.AspNetCore.ResponseCaching/ResponseCachingContext.cs

```diff
@@ -29,21 +30,25 @@ public class ResponseCachingContext
         private TimeSpan _cachedResponseValidFor;
         internal DateTimeOffset _responseTime;
 
-        public ResponseCachingContext(HttpContext httpContext, IResponseCache cache)
-            : this(httpContext, cache, new SystemClock())
+        public ResponseCachingContext(HttpContext httpContext, IResponseCache cache, ResponseCachingOptions options)
+            : this(httpContext, cache, options, new SystemClock())
         {
         }
 
         // Internal for testing
-        internal ResponseCachingContext(HttpContext httpContext, IResponseCache cache, ISystemClock clock)
+        internal ResponseCachingContext(HttpContext httpContext, IResponseCache cache, ResponseCachingOptions options, ISystemClock clock)
```

Comment:
Everywhere else we have the ISystemClock on the Options so you don't need this special constructor.